server/api/router.go

```diff
@@ -282,6 +282,10 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	registerFunc(apiRouter, "/admin/persist-file/{file_name}", adminHandler.SavePersistFile, setMethods("POST"), setAuditBackend(localLog))
 	registerFunc(clusterRouter, "/admin/replication_mode/wait-async", adminHandler.UpdateWaitAsyncTime, setMethods("POST"), setAuditBackend(localLog))
 
+	selfProtectionHandler := newSelfProtectionHandler(svr, rd)
+	registerFunc(apiRouter, "/self_protection/config", selfProtectionHandler.GetSelfProtectionConfig, setMethods("GET"))
```

Comment:
Can we change `self_protection` to `service-middlewares or middlewares`? IMO, self-protection is better to be the project name, not in the code.

Reply:
update